### ISSUE TITLE
Remove WiderContractsApp

### ISSUE THREAD

**Issue by jrossignol** [KSP-CKAN/CKAN#1805]
Now that there's a formal policy in place around this...

Can someone remove Wider Contracts App for me?  The functionality it provides has been moved into Contract Configurator as of version 1.13.0.  The latest version of Wider Contracts App (1.3.4) doesn't do anything other than pop up a message saying it's obsolete.

I'd send a PR on this one, but not sure how this should be handled in the best way so as to not cause trouble to users that have old versions of the mod installed.

**Comment by politas** [comment]
Assuming the recent version of ContractConfigurator is only compatible with the latest version of KSP, we would simply freeze the netkan file and leave the ckan files for people using older versions of KSP. Are you happy with that?

**Comment by politas** [comment]
I like your method of creating a final version that informs users about the change, by the way!

**Comment by jrossignol** [comment]
Thanks!  It was as much in self interest so as to avoid having problems from the same code running twice.

As far as freezing the .netkan - the last version of Wider Contracts App is only compatible with KSP 1.1.3.  So that works, except that the obsolete one sticks around until KSP 1.2 (or 1.1.4) comes out.  In that case would we want to update the last .ckan file to change the display name to say "Obsolete" or something like that?

Basically, I don't want any _new_ users downloading it, but I place a higher priority on old users not getting completely broken.

**Comment by politas** [comment]
If we freeze the netkan, we can change the abstract in the final release ckan to state that.

**Comment by jrossignol** [comment]
Works for me!
